Test/Integration/TestCase.php

```diff
@@ -97,10 +97,10 @@ private function bootstrap()
             return;
         }
 
-        if (version_compare($this->getMagentoVersion(), '2.4.0', '<')) {
-            $this->assertValues = new Magento23();
+        if (version_compare($this->getMagentoVersion(), '2.4.7', '<')) {
```

Comment:
Small observation, but I'm wondering if we should be more precise with our version checks? e.g. 
```suggestion
        if (version_compare($this->getMagentoVersion(), '2.4.6', '>=')  && version_compare($this->getMagentoVersion(), '2.4.7', '<')) {
```
And skip the test if not matching? We have a separate ticket for this so we can circle back later but I was interested in your thoughts?

Reply:
Yes totally ! Good idea :) 
I guess this forces us to update the tests after each Magento minor version release, but it's probably not a bad thing after all.

Reply:
Cool! I agree - every minor release we should be reevaluating our tests vs EOL Magento support. We'll address this on a later ticket. 👍